setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = bosi
-version = 0.0.177
+version = 0.0.179
```

Comment:
I will merge this PR for now, but had a question regarding this VERSION field.

What's the real value of this VERSION here? After we cut 4.0.0, should this be bumped to 4.0.0? Then, can we have this as latest or master?